playbooks/roles/ocp-vpa-operator/README.md

```diff
@@ -67,6 +67,7 @@ Dependencies
  - Role global-secret-update
  - Role set-custom-catalogsource
  - Role golang-installation
+ - To set the go lang version and path please use the variables from the `golang-installation` role. Please make sure, value for `golang_installation_path` is `/usr/local/`.
```

Comment:
IMO this can go as an additional note, instead of dependency.

Reply:
I have added this as a note.